lib/src/authentication/sign_in/sign_in_view_model.dart

```diff
@@ -48,21 +58,46 @@ class SignInViewModel extends BaseViewModel {
       // Disable the form and buttons
       setBusy(true);
 
-      // TODO: Handle login
-      await router.pushReplacement(GlossaryView.routeName);
+      final bool isLoggedIn = await _userRepository.register(
```

Comment:
Should we not use `_userRepository.signIn` here instead?

Reply:
At the end the registration and the sign in are using the same methods. So as the register method was created first, I reused it
So either 
- we move this code to `signIn` and we delete `register`
- we delete `signIn`

Reply:
Let's remove `signIn`, and update the documentation of `register` to explain that. What do you think about renaming the method into `authenticate`or something else ? 🤔 
I find `register` wrong to use for sign in and sign up purpose

Reply:
`authenticate` is fine. Update done

Reply:
So I also updated the `AuthService.signIn...` methods to `authenticate...`